Types incompatible with @types/three@0.163.0

Using `meshline` with `@types/three@0.163.0` and with `skipLibCheck` set to `false` produces the following type error:

```
node_modules/meshline/dist/MeshLineMaterial.d.ts(39,5): error TS2610: 'alphaTest' is defined as an accessor in class 'ShaderMaterial', but is overridden here in 'MeshLineMaterial' as an instance property.
```

This is caused by `@types/three@0.163.0` correctly updating `alphaTest` to be a getter/setter in https://github.com/three-types/three-ts-types/pull/856.

--- a/src/MeshLineMaterial.ts
+++ b/src/MeshLineMaterial.ts
@@ -144,7 +144,6 @@ export class MeshLineMaterial extends THREE.ShaderMaterial implements MeshLineMa
   useAlphaMap!: number
   color!: THREE.Color
   gradient!: THREE.Color[]
-  opacity: number = 1
   resolution!: THREE.Vector2
   sizeAttenuation!: number
   dashArray!: number
@@ -153,7 +152,6 @@ export class MeshLineMaterial extends THREE.ShaderMaterial implements MeshLineMa
   useDash!: number
   useGradient!: number
   visibility!: number
-  alphaTest: number = 0
   repeat!: THREE.Vector2
 
   constructor(parameters: MeshLineMaterialParameters) {

